### ISSUE TITLE
Setup cronjob to log output to STDOUT

### ISSUE THREAD

**Issue by markshust** [mageinferno/docker-magento2-php#10]

**Comment by markshust** [comment]
Tried to modify crontab to following. It executes successfully, but doesn't seem to be piping to stdout.

```
*/1 * * * * su -c "/usr/local/bin/php /srv/www/update/cron.php" -s /bin/sh www-data > /proc/self/fd/2 2>&1
*/1 * * * * su -c "/usr/local/bin/php /srv/www/bin/magento-php cron:run" -s /bin/sh www-data > /proc/self/fd/2 2>&1
*/1 * * * * su -c "/usr/local/bin/php /srv/www/bin/magento-php setup:cron:run" -s /bin/sh www-data > /proc/self/fd/2 2>&1
```

Anyone know why? Fyi, `/proc/self/fd/2` should be stdout; it's the same location used in php-fpm config, which is outputting correctly https://github.com/mageinferno/docker-magento2-php/blob/master/conf/php-fpm.conf#L6

**Comment by markshust** [comment]
Got it! Seems the correct location for crontab output is `/proc/1/fd/2`.

**Comment by markshust** [comment]
Released in `5.6.23-fpm-3` & `7.0.8-fpm`
